src/enums/TemplateLangEnums.js

```diff
@@ -0,0 +1,7 @@
+const templateLang = {
+	PT_BR: 'portuguese-br',
+	EN: 'english',
```

Comment:
Acho que talvez seja mais visual também adicionar a nacionalidade da língua no inglês, segue a sugestão:

```suggestion
	EN_US: 'english-us',
```

Reply:
Eu originalmente tinha feito assim, só que parando para pensar, a versão em inglês que vai estar rodando não seria localizada. Seria para todos os países e não apenas no Inglês Americano. Ai eu acabei deixando assim por hora.